stubs/networkx/networkx/algorithms/components/strongly_connected.pyi

```diff
@@ -0,0 +1,14 @@
+from collections.abc import Hashable, Iterable
+from typing import TypeVar
+
+from networkx.classes.digraph import DiGraph
+from networkx.classes.graph import Graph
+
+_N = TypeVar("_N")
```

Comment:
Same comment as for `Graph[_T]`, could be replaced by using the existing `_Node`